### PR TITLE
CFM-1060: Create view block to display featured groups

### PR DESCRIPTION
**Pull request by DavidHernandez**
Issue: https://issuetracker.amplexor.com/jira/browse/CFM-1060

Desktop:
featured-groups-desktop

Mobile:
featured-groups-mobile

### CODE REVIEW

**Comment by DavidHernandez** [comment]
## Please, merge https://github.com/capacity4dev/capacity4more/pull/1128 first

File: project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.module [hunk 1]
```diff
@@ -93,6 +82,58 @@ function c4m_features_homepage_c4m_suggested_groups_get_groups() {
...
+  return theme('c4m_features_homepage_c4m_featured_groups', $variables);
+}
+
+/**
+ * Loads list of nodes, sorts them and renders them as block list.
+ */
+function c4m_features_homepage_alphabetically_render_node_list($node_ids) {
+  $items = array();
+  foreach ($node_ids as $node) {
+    $items[] = node_load($node->nid);
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.module]: @DavidHernandez 
Can we use `node_load_multiple` instead ?
(better to hit the DB only once)

**Comment by ordavidil** [comment]
@DavidHernandez 
please add a tablet screenshot as well ...

**Comment by DavidHernandez** [comment]
Tablet view:
tablet-featured

**Pull request merged**